application/src/main/webapp/vue-app/components/ExoChatContact.vue

```diff
@@ -192,8 +192,8 @@ export default {
       this.isOnline = false;
     },
     getSpaceURI() {
-      const spaceId = this.name.toLowerCase().split(' ').join('_');
-      getSpaceByPrettyName(this.name).then((space) => {
+      const spaceId = this.name.toLowerCase().split(' ').join('_').split('-').join('');
+      getSpaceByPrettyName(spaceId).then((space) => {
```

Comment:
No need to recalculate the prettyName, it is already a propos of thi scomponent.
```suggestion
      getSpaceByPrettyName(this.prettyName).then((space) => {
```

Reply:
done !